src/components/acciones/repo.jsx

```diff
@@ -0,0 +1,280 @@
+                            disabled={this.state.disabled} />
+                        <label htmlFor='oficina' disabled={this.state.disabled}>Oficina México</label>
+                    </div>
+                    <div>
+                        <input
+                            type='radio'
+                            id='dealer'
+                            value='Dealer'
+                            checked={respuestas.lugarCustodia !== 'Oficina México'}
+                            onChange={this.handleRadioChange.bind(this, 'lugarCustodia')}
+                            disabled={this.state.disabled} />
+                        <label htmlFor='dealer' disabled={this.state.disabled}>Dealer</label>
+                    </div>
+                    {this.renderInput()}
+                </div>
+                <div className='element-wrapper'>
+                    <h5 className='text-label'>Fechas de Subasta</h5>
+                    {this.renderFechaSubasta(1)}
+                    {this.renderFechaSubasta(2)}
+                    {this.renderFechaSubasta(3)}
+                    {this.renderFechaSubasta(4)}
+                    {this.renderFechaSubasta(5)}
+                    {this.renderAddDateButton()}
+                </div>
+                <div className='element-wrapper'>
+                    <h5 className='text-label'>Monto de Venta</h5>
+                    <input
+                        type='text'
+                        value={this.state.respuestas.montoVenta}
+                        onChange={this.handleChange.bind(this, 'montoVenta')}
+                        disabled={this.state.disabled} />
+                </div>
+                <div className='element-wrapper'>
+                    <h5>Fecha de Venta</h5>
+                    <DateSelect date={this.state.respuestas.fechaVenta} onChange={this.handleFechaChange.bind(this, 'fechaVenta')} />
+                </div>
+                {this.renderComentarios()}
+                {this.renderButton()}
+            </div>
+        );
+    },
+    renderFechasVoBo: function () {
+        if (this.state.respuestas.personal === 'Personal') {
+            return;
+        }
+
+        return (
+            <div>
+                <div className='element-wrapper'>
+                    <h5>Fecha VoBo RBU</h5>
+                    <DateSelect date={this.state.respuestas.fechaVoBoRbu} onChange={this.handleFechaChange.bind(this, 'fechaVoBoRbu')} />
+                </div>
+                <div className='element-wrapper'>
+                    <h5>Fecha VoBo GMF</h5>
+                    <DateSelect date={this.state.respuestas.fechaVoBoGmf} onChange={this.handleFechaChange.bind(this, 'fechaVoBoGmf')} />
+                </div>
+            </div>
+        );
+    },
+    renderFechaSubasta: function (num) {
+        if (!this.state.respuestas['fechaSubasta' + num] && this.state.respuestas['fechaSubasta' + num] !== null) {
```

Comment:
esto de aquí sí funciona; pero no entiendo por qué el `…&& this.state.respuestas['fechaSubasta' + num] !== null`